automation_tools/satellite6/upgrade/satellite.py

```diff
@@ -43,7 +43,9 @@ def satellite6_setup(os_version):
             print('The following environment variable(s) must be set in jenkin'
                   'environment: {0}.'.format(', '.join(missing_vars)))
             sys.exit(1)
-        sat_image = os.environ.get('RHEV_SAT_IMAGE')
+        os.environ['RHEV_SAT_IMAGE'] = 'qe-sat6-upgrade-rhel6-fixed-sat{}'
```

Comment:
These lines does not make sense to me, I think the following line will be the same:

```python
sat_image = 'qe-sat6-upgrade-rhel6-fixed-sat{}'.format(
    os.environ.get('FROM_VERSION').replace('.', '')
)
```

Using replace will raise an exception if the env var is not available.